### ISSUE TITLE
Spoon plugin can only be applied to android application or library projects

### ISSUE THREAD

**Issue by OliverCulleyDeLange** [jaredsburrows/gradle-spoon-plugin#95]
I'm wondering why this is the case? Is there a technical reason, or is it just unimplemented?

I have an external test project setup using the `com.android.test` plugin, and therefor i can't apply this lovely plugin. 

Happy to attempt a PR if its possible...

**Comment by OliverCulleyDeLange** [comment]
Code [here ](https://github.com/jaredsburrows/gradle-spoon-plugin/blob/b9f5afa843e740d5737c3964018d780aad73d54f/src/main/kotlin/com/jaredsburrows/spoon/SpoonPlugin.kt#L68)for reference

**Comment by jaredsburrows** [comment]
@OliverCulleyDeLange Do you have `androidTests` in your `com.android.test` project? 

I found an example project here: https://github.com/googlesamples/android-testing-templates. 

I can look into adding support for this.

**Comment by OliverCulleyDeLange** [comment]
Yes, i have androidTests in the sense that i have end to end instrumented tests in the project, however they're not in  the androidTest source set, they're simply in src/main/java/com.blah...

I also used that example project to set it up, using [this](https://github.com/googlesamples/android-testing-templates/blob/master/AndroidTestingBlueprint-kotlinApp/module-flavor1-androidTest-only/build.gradle) as an example.
